cmd/validator/credentials/set/process.go

```diff
@@ -160,14 +160,28 @@ func (c *command) generateOperationFromMnemonicAndPath(ctx context.Context) erro
 	}
 
 	validatorKeyPath := c.path
+	// err if no path is provided
+	if validatorKeyPath == "" {
```

Comment:
This function is only called if `c.path` is non-nil, so this check is not necessary.

Reply:
Done. I also removed the `TestGenerateOperationFromMnemonicAndPath/NoPathProvided` I added since it is not needed.